lib/library.rb

```diff
@@ -0,0 +1,46 @@
+require 'book'
+require 'date'
+
+class Library
+   attr_accessor :books   #TODO duedate person status
+
+   def initialize(library)
+    @books = []
+   end
+
+   def check_in(book)
+    add = {item: book, status: :okay, person: :non, duedate: " " }
+    @books << add
+   end
+
+   def lend(title)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for lend is too high. [26.17/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [22/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)